annotator-core/src/main/java/edu/ucr/cs/riple/core/metadata/trackers/ParameterRegionTracker.java

```diff
@@ -62,7 +62,8 @@ public Optional<Set<Region>> getRegions(Location location) {
             .collect(Collectors.toSet());
     // Add the method the fix is targeting.
     regions.add(new Region(parameter.clazz, parameter.method));
-    // Add all call sites.
+    // Add all call sites. It will also reserve call sites to prevent callees have their parameters
+    // annotated simultaneously. See test: CoreTest#nestedParameters.
```

Comment:
I extracted the test case and the comment to #138. The main issue is that, I wanted to emphasize for parameter, the call sites of the method, can be a potentially impacted region.

Just to avoid scenario below:

```java
void foo(Object p){
       bar(p);
}

void bar(Object p)
```

assume that we have two fixes on foo#bar and `bar#p` and `foo#p`, while we are analyzing impact of making `foo#p` nullable, `bar#p` should not be annotated as nullable, otherwise we will miss the triggered error that p is nullable while invoking `bar`. This addition in effective region will make a region conflict for `foo#p` and `bar#p` and they will not be processed simultaneously.